cloud/docker/docker_container.py

```diff
@@ -1557,7 +1560,7 @@ def __init__(self, client):
         self.facts = {}
 
         state = self.parameters.state
-        if state in ('started', 'present'):
+        if state in ('stopped', 'started', 'present'):
```

Comment:
@chouseknecht as far as i can tell, this isn't quite correct.

situation: if a user sends `state: stopped` for a container that doesn't exist, `present(self, state)` will create it and return (see lines 1581+) 

because if the container does not exist, `present(self, state)` will create it on line 1581.